From c87b68f400dc13674647e44372dff6c2288009b3 Mon Sep 17 00:00:00 2001
From: Jean-Christophe Fillion-Robin <jchris.fillionr@kitware.com>
Date: Tue, 18 Jan 2022 03:23:36 -0500
Subject: [PATCH 2/4] MinGW support: Add patches-win32/02-signalmodule.patch

This commit integrates changes originally added to the
python-cmake-buildsystem project as 18d49757e (Mingw32 support,
and add cmake options for disabling)

Note that in more recent version of Python, this patch is obsoleted
by the following commits:

- See python/cpython@7faf70512 (introduced in 3.4)
      Issue #17591: Use lowercase filenames when including Windows header files.

Co-authored-by: David Sansome <me@davidsansome.com>
---
 Misc/ACKS              | 1 +
 Modules/signalmodule.c | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Misc/ACKS b/Misc/ACKS
index 624b6164be..a555b1b660 100644
--- a/Misc/ACKS
+++ b/Misc/ACKS
@@ -653,6 +653,7 @@ Gabriel de Perthuis
 Tim Peters
 Benjamin Peterson
 Chris Petrilli
+Roumen Petrov
 Bjorn Pettersen
 Geoff Philbrick
 Gavrie Philipson
diff --git a/Modules/signalmodule.c b/Modules/signalmodule.c
index 908c2ee994..e2b6ee04f6 100644
--- a/Modules/signalmodule.c
+++ b/Modules/signalmodule.c
@@ -7,7 +7,7 @@
 #include "intrcheck.h"
 
 #ifdef MS_WINDOWS
-#include <Windows.h>
+#include <windows.h>
 #ifdef HAVE_PROCESS_H
 #include <process.h>
 #endif
-- 
2.22.0

